scripts/upgrade

```diff
@@ -158,7 +158,8 @@ last_version=$next_version
 datadir="/home/yunohost.app/${app}/data"
 
 # Set write access for the following commands
-chown -R $app: "$final_path" "$datadir"
+chown -R $app: "$final_path" 
+find ${datadir}/ \! -user $app -o \! -group $app -exec chown $app: {} \;
```

Comment:
It could use some explanation.

Reply:
datadir can have many files, make chown on any take time because it's a write operation on any file.
Checking if file don't have the right owner can reduce this time.

Reply:
"It" not "I"
I don't need any explanations about bash. But this code should be commented.

Reply:
Done ! (sorry for my english...)